conf/application.conf

```diff
@@ -34,15 +34,11 @@ application.session.httpOnly = true
 play.i18n.langCookieHttpOnly = true
 play.i18n.langs = ["en", "cy"]
 
-urBannerToggle = true
+urBannerToggle = false
```

Comment:
references to this flag have been removed in the AppConfig. This flag now looks like its being set in the Main class. Is this a config value we can now get rid of if that's the case